engine/classes/Elgg/Database.php

```diff
@@ -248,13 +252,14 @@ public function insertData($query) {
 	 *
 	 * @note Altering the DB invalidates all queries in the query cache.
 	 *
-	 * @param string $query      The query to run.
-	 * @param bool   $getNumRows Return the number of rows affected (default: false)
+	 * @param string $query        The query to run.
+	 * @param array  $params       Query params. E.g. [1, 'steve'] or [':id' => 1, ':name' => 'steve']
+	 * @param bool   $get_num_rows Return the number of rows affected (default: false).
 	 *
 	 * @return bool|int
 	 * @throws \DatabaseException
 	 */
-	public function updateData($query, $getNumRows = false) {
+	public function updateData($query, array $params = [], $get_num_rows = false) {
```

Comment:
Hmm, `elgg()->getDb()` is technically public API, so this whole class is kinda frozen in 2.x. :-1: